pkg/app/app.go

```diff
@@ -120,8 +121,9 @@ func (a *App) Repos(c ReposConfigProvider) error {
 func (a *App) DeprecatedSyncCharts(c DeprecatedChartsConfigProvider) error {
 	return a.ForEachState(func(run *Run) (_ bool, errs []error) {
 		err := run.withPreparedCharts("charts", state.ChartPrepareOptions{
-			SkipRepos: true,
-			SkipDeps:  true,
+			SkipRepos:   true,
+			SkipDeps:    true,
+			Concurrency: 2,
```

Comment:
@stek29 why 2?

Reply:
because it was hard coded to 2 before, and i didn’t add Concurrency option to this command.